src/interfaces/config.ts

```diff
@@ -24,6 +26,51 @@ export type VersionDetails = {
   shell?: string
 }
 
+export interface Theme {
+  alias?: Color
+  bin?: Color
+  command?: Color
+  commandSummary?: Color
+  dollarSign?: Color
+  flag?: Color
+  flagDefaultValue?: Color
+  flagOptions?: Color
+  flagRequired?: Color
+  flagSeparator?: Color
+  flagType?: Color
+  sectionDescription?: Color
+  sectionHeader?: Color
+  topic?: Color
+  version?: Color
+}
+
+// TODO: find a way to create this array at build time based on the interface keys
+export const THEME_KEYS = [
+  'alias',
+  'bin',
+  'command',
+  'commandSummary',
+  'dollarSign',
+  'flag',
+  'flagDefaultValue',
+  'flagOptions',
+  'flagRequired',
+  'flagSeparator',
+  'flagType',
+  'sectionDescription',
+  'sectionHeader',
+  'topic',
+  'version',
+]
+
+export function parseTheme(untypedTheme: Record<string, string>): Theme {
```

Comment:
This doesn't belong in the `interfaces` directory. Should go somewhere under `utils`

Reply:
fixed